From 8c2b2f1d7df9a38fee13f4a8d99f4a08d721b75c Mon Sep 17 00:00:00 2001
From: Mauro Rossi <issor.oruam@gmail.com>
Date: Sun, 28 Feb 2021 02:01:38 +0100
Subject: [PATCH 086/100] x86: Kconfig: replace '---help---' with 'help' (v2)

Fixes the following building errors happenign with kernel 5.9:

drivers/platform/x86/Kconfig:461: syntax error
drivers/platform/x86/Kconfig:460: unknown statement "---help---"
...
drivers/platform/x86/Kconfig:762: syntax error
drivers/platform/x86/Kconfig:761: unknown statement "---help---"

(v2)
Changes due to commit:
f1e1ea516721 ("platform/x86: Move all dell drivers to their own subdirectory")
---
 drivers/platform/x86/Kconfig      | 2 +-
 drivers/platform/x86/dell/Kconfig | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/platform/x86/Kconfig b/drivers/platform/x86/Kconfig
index 91b1962704d4..773783a47ad3 100644
--- a/drivers/platform/x86/Kconfig
+++ b/drivers/platform/x86/Kconfig
@@ -620,7 +620,7 @@ config S103T_WMI
 	tristate "Lenovo S10-3T Rotate Button Support"
 	depends on ACPI_WMI
 	depends on INPUT
-	---help---
+	help
 	  This is a driver for the Lenovo ideapad S10-3T rotate button.
 
 	  To compile this driver as a module, choose M here: the module will
diff --git a/drivers/platform/x86/dell/Kconfig b/drivers/platform/x86/dell/Kconfig
index dcb42d771bdd..27ad99e4a014 100644
--- a/drivers/platform/x86/dell/Kconfig
+++ b/drivers/platform/x86/dell/Kconfig
@@ -207,7 +207,7 @@ config DELL_WMI_SYSMAN
 config DV11P_TABLET
 	tristate "Dell Venue 11 Pro Tablet support"
 	depends on ACPI
-	---help---
+	help
 	  This driver provides power button support and docking events for the Dell Venue 11 Pro tablet.
 
 endif # X86_PLATFORM_DRIVERS_DELL
-- 
2.17.1

